packages/big-design/src/components/Checkbox/styled.tsx

```diff
@@ -32,25 +32,32 @@ export const StyledCheckbox = styled.label<StyledCheckboxProps>`
     checked || isIndeterminate ? theme.colors.primary : theme.colors.secondary30};
   border-radius: ${({ theme }) => theme.borderRadius.normal};
   color: ${({ theme }) => theme.colors.white};
+  cursor: pointer;
   display: inline-flex;
   height: ${({ theme }) => theme.spacing.large};
   justify-content: center;
-  transition: all 150ms;
+  transition: all 150ms ease-out;
```

Comment:
I know this was done before this PR, but could you explicitly set the transition instead of using `all`? 
```suggestion
  transition: 150ms ease-out;
  transition-property: border-color, box-shadow, color, visibility;
```